README.md

```diff
@@ -29,7 +29,7 @@ Core features include:
 # Documentation
 
 - The `talawa` documentation can be found [here](https://palisadoesfoundation.github.io/talawa-docs/).
-- Want to contribute? Look at [CONTRIBUTING.md][link] to get started.
+- Want to contribute? Look at [CONTRIBUTING.md][https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md] to get started.
```

Comment:
Remember the syntax for a hyperlink in markdown is

`[LinkTitle](URL)`

e.g) `[CONTRIBUTING.md](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)`

gives

[CONTRIBUTING.md](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)